person.rb

```diff
@@ -3,7 +3,7 @@
 require_relative './trimmer'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
   attr_accessor :name, :age
```

Comment:
- [ ] the user can read and write `rentals`, kindly make sure to add it to the atter_accessor instead